gapic/templates/%namespace/%name_%version/%sub/services/%service/async_client.py.j2

```diff
@@ -361,7 +367,16 @@ class {{ service.async_client_name }}:
 
         # Wrap the RPC method; this adds retry and timeout information,
         # and friendly error handling.
-        rpc = self._client._transport._wrapped_methods[self._client._transport.{{ method.transport_safe_name|snake_case }}]
+        if raw_response_callback:
+            if "with_call" in inspect.signature(gapic_v1.method_async.wrap_method).parameters:
```

Comment:
Is this expensive? Seems like it could add quite a bit of latency to each call

Maybe directly check the __version__?

Reply:
Good catch!